### ISSUE TITLE
How to use this library ?

### ISSUE THREAD

**Issue by ghost** [yutter/ArduinoJWT#8]
I want use this library to encode information in arduino uno and send it to the web server , can you help me ? ( I want an example code for encoding )

**Comment by guilleAnselmi** [comment]
![image](https://user-images.githubusercontent.com/30878097/44377139-e6acbb00-a4d1-11e8-98b5-868e4a936bef.png)

**Comment by Archeresque** [comment]
Take a look at the arduinojwt.h file. There's not many functions so it's fairly straightforward and it explains what you should call the function and the data types the function accepts. In the issues forum, there is an early thread where someone writes a quick straightforward example.

**Comment by mer30boy** [comment]
For example, we have the following payload:

{ 'id':'1','fname':'john','lname':'snow'}

Please give an example to encode and decode this payload by 'arduinojwt'

**Comment by chrismoorhouse** [comment]
Did you look at the source? It is pretty simple.

Create an instance of ArduinoJWT class passing in your psk then call encodeJWT passing in the JSON to encode it and decodeJWT passing in the jwt to get the payload back

**Comment by DprasK** [comment]
ArduinoJWT usage:

#include <ArduinoJWT.h>

char key[] = "tester";
ArduinoJWT jwt(key);

void setup() {
  String a, tes;
  tes = "testing";
  a = jwt.encodeJWT(tes);
  Serial.println(a); //result of encryption
  jwt.decodeJWT(a, tes);
  Serial.println(tes);
}

@chrismoorhouse, i advise you to make like this in readme.md

**Comment by chrismoorhouse** [comment]
I think that looking at the header file makes it pretty simple to use 😊

**Comment by DprasK** [comment]
yeah, as your type this simple use. but i have to read ArduinoJWT.cpp

thank you 😊
